src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,19 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class Lottos {
+    public static int PRICE_PER_LOTTO = 1000;
+
+    private List<Lotto> lottos;
+
+    public Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
```

Comment:
```suggestion
        this.lottos = new ArrayList(lottos);
```
이렇게 감싸면 null로 부터 안전할거에요